chrome/script.browserify.js

```diff
@@ -195,14 +195,19 @@ function fillLoginForm(login) {
 	}
 
 	update(field('input[type=password]'), ${JSON.stringify(login.p)});
-	update(field('input[type=email], input[type=text], input:first-of-type'), ${JSON.stringify(login.u)});
+	// Guess username field
+	var usernameFields = field('input[id=username], input[id=login], input[id=email]');
```

Comment:
you'd get the first match, in case of the AWS console, it's still the account field.